src/Functions/FunctionBase64Conversion.h

```diff
@@ -100,7 +100,7 @@ class FunctionBase64Conversion : public IFunction
     DataTypePtr getReturnTypeImpl(const ColumnsWithTypeAndName & arguments) const override
     {
         FunctionArgumentDescriptors mandatory_arguments{
-            {"value", &isStringOrFixedString<IDataType>, nullptr, "String or FixedString"}
+            {"value", static_cast<FunctionArgumentDescriptor::TypeValidator>(&isStringOrFixedString), nullptr, "String or FixedString"}
```

Comment:
Since the usage of `FunctionArgumentDescriptor::TypeValidator` is so widespread, should we give it a shorter name?